### ISSUE TITLE
Normalized weapon damage

### ISSUE THREAD

**Issue by Ostaroar** [helgemahrt/BackstabCalculator#1]
Not sure if I'm right ... could it be, that you do not use normalized weapon damage for backstab, sinister strike, etc.?

https://wowwiki-archive.fandom.com/wiki/Normalization

**Comment by helgemahrt** [comment]
That's correct. I created this add-on for Vanilla, to make maximizing backstab damage easier. Damage normalization makes this a bit redundant - you just look at the higher DPS weapon and go with it. (It took all the fun out of backstab :D )

**Comment by Vanillabro** [comment]
I this addon accurate any longer?  the 1.0.8 vs 1.0.9 have two VERY different delves.  I used 1.0.8 in Classic and it seemed make sense, but the next patch is almost backwards.  Is this due to normalization?

**Comment by helgemahrt** [comment]
As you can see, I did not update this add-on for years :) As I said before, with normalization, this add-on loses its value because you just need to look at the DPS value to know which dagger does more (backstab) damage.
